masterfabric-mobile/swift-camp
# CL - TextField Viper Implementation

## 📋 PR Description

This PR implements a new TextFieldView built using VIPER architecture. The view includes four types of textfields. First one is only accept integer numbers, second one is accept decimal, third one accept everything fourth and five one also accept everything but they have a leading and a suffix icon. The UI also includes a back button to navigate to the HomeView. 

---

## ✅ Checklist


---

## 🛠 Steps to Test

1. Open the app in Xcode
2. Run the app with simulator
3. Click the "Go to TextFields" button
4. Start typing to textfields and ensure that typing only characters which they accept
---

## 🔗 Related Links

- Documentation: https://jainrishu111.medium.com/textfield-in-swiftui-869a6f44aa63

---

### 📷 Screenshots (Optional)
![image](https://github.com/user-attachments/assets/e6326c62-d03d-4e2f-9670-642fcf0d7efe)![image](https://github.com/user-attachments/assets/17e631ed-0e82-4d14-a983-da81d4fd5a07)![image](https://github.com/user-attachments/assets/88a52817-c362-4236-aa0e-62057b0a17d2)![image](https://github.com/user-attachments/assets/e19e5f1f-db65-4dcd-8e0f-cfdcb78ecdb8)
